i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chat-backup-verification.mdx

```diff
@@ -0,0 +1,119 @@
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Chat-Backup Verifikation</h2>
+    <p class="hero__subtitle">Wie Sie eine Sitzung mittels QR-Verifikation validieren und was Sie tun können, falls Ihr Gerät den QR-Code nicht scannen kann oder keine anderen Geräte verfügbar sind.</p>
+  </div>
+</div>
+
+## QR-Verifikation
+
+Im Login-Prozess werden Sie automatisch zur Qr-Code Verifikation weitergeleitet, sofern mindestens eines Ihrer Geräte ein Mobilgerät ist. 
+Falls Sie dennoch stattdessen zur Zahlen- oder Emoji-Verifikation weitergeleitet werden, prüfen Sie bitte, ob für eines Ihrer Geräte ein Update verfügbar ist. 
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Klicken Sie in der <b>Verifikationsanfrage</b> auf <b>Fortfahren</b>, um die Bestätigungsanfrage anzunehmen.</li>
+    <li>Falls das andere Gerät ein Mobilgerät ist, wird Ihr Browser Tab nun einen QR-Code anzeigen. Andernfalls werden Sie gebeten, stattdessen Zahlen zu vergleichen.</li>
+    <li>Scannen Sie nun den QR-Code, der in Ihrem Browser angezeigt wird mit Ihrem Mobilgerät. Falls beide Geräte Web-Browser sind, vergleichen Sie bitte die Zahlen und bestätigen Sie diese.</li>
+    <li>Falls Sie den QR-Code aus irgendeinem Grund nicht scannen können, tippen Sie bitte auf Ihrem Mobilgerät auf <b>Zahlen vergleichen statt QR-Code</b>.</li>
+    <li>Nachdem Sie den QR-Code gescannt haben, werden Sie gefragt, ob wirklich Sie selbst den QR-Code gescannt haben. Klicken Sie auf <b>Ja, das war ich</b> um die Verifizierung durchzuführen.</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobil">
+  <ol>
+    <li>Tippen Sie in der <b>Verifikationsanfrage</b> auf <b>Fortfahren</b>, um die Bestätigungsanfrage anzunehmen.</li>
+    <li>Eines Ihrer Geräte zeigt jetzt einen QR-Code an, das andere Gerät öffnet einen QR-Codescanner.</li>
+    <li>Bitte scannen Sie den QR-Code.</li>
+    <li>Falls Sie aus irgendeinem Grund nicht in der Lage sind, den QR-Code zu scannen, tippen Sie auf <b>Zahlen vergleichen statt QR-Code</b>.</li>
+    <li>nach dem Scannen des QR-Codes, werden Sie gefragt, ob es wirklich Sie waren, die den QR-Code auf Ihrem bereits aktiven Gerät gescannt. Tippen Sie auf <b>Ja, das war ich</b> um die Überprüfung abzuschließen.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+
+## Zahlen-Verifikation
+
+Falls beide Geräte Desktop Geräte oder Web Browser sind werden Sie automatisch anstelle der QR-Verifikation zur Zahlen-Verifikation aufgefordert. Vergleichen Sie einfach die Zahlen und klicken Sie auf <b>Stimmt überein</b>.  
+
+### Zahlen-Verifikation auf Mobilgeräten verwenden
+
+Möglicherweise müssen Sie auch die Zahlen-Verifikation nutzen, wenn z.B. die Kamera Ihres Mobilgerätes defekt ist. Sollte dies der Fall sein, können Sie die folgenden Schritte befolgen. 
```

Comment:
Diesen Satz gefällt mir auch gut, könnte man auch oben einfügen

Reply:
Was hältst du davon, in der QR-Verifikation einen Link zu dem Block (Zahlenverifikation) einzufügen.
Also so: "Falls Sie den QR-Code nicht scannen können, folgen Sie bitte der Anleitung zur **Zahlen-Verifikation**." (Zahlen-Verifikation dann als Link)